app/controllers/user_controller.rb

```diff
@@ -51,9 +56,19 @@ def index
         if "ACCEPTING".casecmp? event.status
           acceptingTableData << object
         end
+        if "ACCEPTING".casecmp?(event.status) && event.statename == user_state
```

Comment:
duplicate if condition. could add as nested condition

Reply:
It is not duplicated as it is two different lists that are being created. The first if checks if valid for the general list of events and the second generated reccomendations